release/Makefile

```diff
@@ -78,46 +77,62 @@ go-unpack: $(GO_UNPACKED)
 
 # Cross-compile Go package for target platform
 $(GO_DIR)/go/pkg/%/.built: $(GO_UNPACKED)
+	rm -rf gopath/pkg/$*
 	$(call msg,"Building Go for $*")
-	cd $(GO_DIR)/go/src && $(FLAGS_$*) ./make.bash --no-clean >/dev/null 2>&1
-	touch $@
+	cd $(GO_DIR)/go/src && $(FLAGS_$*) ./make.bash --no-clean >lastbuild.log 2>&1
+	touch $(GO_DIR)/go/pkg/$*/.built
 
 go-compile: $(foreach PLATFORM,$(PLATFORMS),$(GO_DIR)/go/pkg/$(PLATFORM)/.built)
 .PHONY: go-compile
 
 
 # Cross-compile final applications
-out/$(RELEASE_VERSION)/%/websocketd: $(GO_DIR)/go/pkg/%/.built $(wildcard ../*.go) go-path/src/github.com/joewalnes/websocketd
+out/$(RELEASE_VERSION)/%/websocketd out/$(RELEASE_VERSION)/%/websocketd.exe: $(GO_DIR)/go/pkg/%/.built $(wildcard ../*.go) go-path/src/github.com/joewalnes/websocketd
```

Comment:
This does not work with %  and I left it with the extension typed to make rule. This seem to work for all platforms and builds correct file in different subdirs of out/RELEASEVER